public/css/index.css

```diff
@@ -0,0 +1,44 @@
+/**
+* Styles for index.html and its components.
+**/
+
```

Comment:
You should alphabetize the selectors and the styles
`tbody` should go before `thead`